tests/commands/test_store_api.py

```diff
@@ -150,6 +150,10 @@ def test_not_logged_in_warns_alternate_auth(monkeypatch):
 
 def test_login(client_mock, config):
     """Simple login case."""
+    # set up a response from client's login
+    acquired_credentials = "super secret stuff"
```

Comment:
maybe for _interface_ clarity make this a base64 encoded string.

Reply:
Actually charmcraft does not care about it. It may be a zip binary in the future. It's really obscure to Charmcraft...